lib/features/push_notification/presentation/services/fcm_service.dart

```diff
@@ -35,6 +38,10 @@ class FcmService {
     backgroundMessageStreamController.add(newRemoteMessage);
   }
 
+  void handleRefreshToken(String newToken) {
+    fcmTokenStreamController.add(newToken);
+  }
+
   void _closeStream() {
     foregroundMessageStreamController.close();
     backgroundMessageStreamController.close();
```

Comment:
should add close for `fcmTokenStreamController` also?

Reply:
Oh, i closed it.